sycl/test/CMakeLists.txt

```diff
@@ -94,9 +100,10 @@ if(SYCL_BUILD_PI_CUDA)
     PARAMS "SYCL_BE=PI_CUDA"
     DEPENDS ${SYCL_TEST_DEPS}
```

Comment:
should check-sycl-cuda also be excluded from check-all?

Reply:
I'm not sure what is the reason to exclude any of the tests from `check-all`, but I aligned all target configurations.